docs/desktop/display/installing_nvidia_gpu_drivers.md

```diff
@@ -10,7 +10,7 @@ Nvidia is one of the most popular GPU manufacturers. You can install Nvidia GPU
 
 !!! Note
 
-    The link for pre-installation actions in Nvidia's official guide is broken. To install the Nvidia driver, you must install the necessary utilities and dependencies from their official repository. 
+    The link for pre-installation actions in Nvidia's official guide is broken. To install the Nvidia driver, you must install the necessary utilities and dependencies from their official repository.
```

Comment:
Nvidia or NVIDIA ?
https://www.nvidia.com/fr-fr/

Reply:
Good question. The rule is that if it is the product name, then all capitals is preferred. If it is the generic term, then lowercase is fine. In this case, it probably should be capitalized for "Nvidia's official guide" but lower case for "Nvidia driver" line. Go ahead and push another PR if you want to fix it. I had to Google the anwer. As far as NVIDIA is concerned, we probably should not only make it capital in the first instance, but add a trademark symbol. [see here for how to do that](https://docs.rockylinux.org/guides/contribute/rockydocs_formatting/#superscript-subscript-and-special-symbols)

Reply:
@sspencerwire  please review !

Reply:
Indeed, during the translation I saw that the problem of spelling NVIDIA/Nvidia is not so simple, see e.g.
https://en.wikipedia.org/wiki/Nvidia
and
https://en.wikipedia.org/wiki/Nvidia_GTC